### DISCUSSION TITLE
欢迎来到 textdata.cn 留言区

### DISCUSSION THREAD

**Discussion by hiDaDeng** [hiDaDeng/hidadeng.github.io#31] [交流]
该留言板由giscus实现， 托管在github， 免费的！所以界面并不美观，能实现基本的问答、社交等功能， 对咱们帮助已经很大了。 毕竟github+giscus免费咱还要啥自行车。

希望这里能你能学有所得，希望在这里能认识到更多同行之人。

**Comment by hiDaDeng** [comment]
giscus支持markdown，进入 [留言区链接 https://github.com/hiDaDeng/hidadeng.github.io/discussions/](https://github.com/hiDaDeng/hidadeng.github.io/discussions/)， 留言内容中可添加链接、文件附件、添加图片，且能实现点击图片跳转

[iris.csv](https://github.com/hiDaDeng/hidadeng.github.io/files/15344417/iris.csv)

<br>

[![python实证指标构建与文本分析](https://github.com/hiDaDeng/hidadeng.github.io/assets/18514610/f861e304-df96-4b4e-83be-e8fb218c7b88)](https://textdata.cn/blog/management_python_course/)

<br>

[![b站](https://github.com/hiDaDeng/hidadeng.github.io/assets/18514610/613afa39-d48e-430d-b06b-7fbe2ce70e46)](https://www.bilibili.com/video/BV1se4y1C7MV/)

**Comment by Wolfe-Jam** [comment]
🎉 **Update: Additional Milestone Since Posting**

`.faf` is now **IANA-registered** as `application/vnd.faf+yaml`

This makes claude-faf-mcp:
- Built on an official Internet-standard format
- Only persistent project context MCP server with IANA-registered foundation
- Increased long-term stability and interoperability

Full details: https://www.faf.one/blog/iana-registration

Still curious about the github.com/mcp curation process and happy to provide any additional info that might help with visibility!
